$OpenBSD: patch-openbsd_OpenBSDProcessList_h,v 1.3 2020/12/08 16:23:17 sthen Exp $

Don't include "offline" CPUs in the stats (i.e. CPUs which have been
disabled by sysctl hw.smt=0).

Index: openbsd/OpenBSDProcessList.h
--- openbsd/OpenBSDProcessList.h.orig
+++ openbsd/OpenBSDProcessList.h
@@ -36,6 +36,7 @@ typedef struct OpenBSDProcessList_ {
 
    CPUData* cpus;
 
+   int *cpuIndex;
 } OpenBSDProcessList;
 
 
